chapter2/tests/t3/chapter2-t3-velero-eks-diff.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Task 3 - find namespaces without the backup
+set -eo pipefail
+
+# Downloading YAML, extracting already saved namespaces into array.
+# Here we are using the from_yaml filter to convert a multi-line string into a YAML data structure,
+# after which we can use the dot notation to access the value of includedNamespaces.
+# See my explanation in the PR 
+savedns=($(curl -sSL https://gist.github.com/dmitry-mightydevops/016139747b6cefdc94160607f95ede74/raw/velero.yaml \
+  | yq eval '.spec.source.helm.values | from_yaml | .schedules.system.template.includedNamespaces[]' ))
```

Comment:
We always try to avoid `eval` usage in our scripts to avoid security issues - https://www.tutorialspoint.com/why-should-eval-be-avoided-in-bash-and-what-should-i-use-instead

Basically there can be passed any code to eval, which can be executed by your script, i.e. Dmitry can add any code to `.spec.source.helm.values..schedules.system.template.includedNamespaces[]`, which you can then execute by your script run on your local computer (so that could be some `rm -rf` code or any other code which can break smth in your local computer)

Please find a way to do this without `eval`